bftengine/src/bftengine/SimpleClientImp.cpp

```diff
@@ -187,15 +188,16 @@ int SimpleClientImp::sendRequest(uint8_t flags,
                                  uint64_t timeoutMilli,
                                  uint32_t lengthOfReplyBuffer,
                                  char* replyBuffer,
-                                 uint32_t& actualReplyLength) {
+                                 uint32_t& actualReplyLength,
+                                 std::string cid) {
   bool isReadOnly = flags & READ_ONLY_REQ;
   bool isPreProcessRequired = flags & PRE_PROCESS_REQ;
-
+  std::string msgCid = cid.empty() ? std::to_string(reqSeqNum) : cid;
   // TODO(GG): check params ...
-  LOG_DEBUG(GL,
-            "Client " << clientId_ << " - sends request " << reqSeqNum << " (isRO=" << isReadOnly
-                      << ", isPreProcess=" << isPreProcessRequired << " , request size=" << lengthOfRequest
-                      << ", retransmissionMilli=" << limitOfExpectedOperationTime_.upperLimit() << " ) ");
+  LOG_INFO(GL,
```

Comment:
Do we really want to log every single client request? I think we demoted this on purpose.

Reply:
Fixed